qualtran/Adjoint.ipynb

```diff
@@ -26,7 +26,9 @@
     "from qualtran.drawing import show_bloq\n",
     "from qualtran.bloqs.basic_gates import ZeroState\n",
     "\n",
+    "print(ZeroState())\n",
```

Comment:
remove?

Reply:
this is on purpose to show that it turns into "ZeroEffect"